dev/docs/dartdoc_options.yaml

```diff
@@ -0,0 +1,3 @@
+dartdoc:
+  ignore:
+    - no-documentable-libraries
```

Comment:
Can you leave a comment here explaining why we turned this off?

Reply:
Done.